pkg/cmd/proxy.go

```diff
@@ -90,7 +98,9 @@ func setupProxyTunnel(opts *proxyOpts) error {
 
 	fmt.Println("SSH tunnel started, please open another terminal and setup environment")
 	fmt.Printf("export HTTPS_PROXY=http://%s\n", opts.ListenAddr)
-	return server.ListenAndServe()
+
+	err = server.ListenAndServe()
+	return err
```

Comment:
The change here is no changing to good anything, please return back the `return server.ListenAndServe()`

Reply:
Nice catch! I somehow thought it would be needed for `defer tunn.Close()` to work. :see_no_evil: